### PR TITLE
fix: simplify sponsor box

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5462

scrnli_11_3_2020_11-07-59 AM

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by mariobehling — CHANGES_REQUESTED** [review]
Screenshot from 2020-11-03 11-46-56

* Please use the same spacing to the box edge for the "Edit" button as in other boxes
* Please decrease the space between previous/next button and the above table

File: app/templates/components/events/view/overview/event-sponsors.hbs [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <LinkTo
   @route="events.view.edit.sponsors"
   @tagName="button" class="ui right floated blue button">
-    {{t 'Add Sponsors'}}
+    {{t 'Edit'}}
   </LinkTo>
   <div class="header">{{t 'Event sponsors'}}</div>
 </div>
-<div class="content">
+<div class="content" style="width:94%;margin-left:3%;">
```
Comment (iamareebjamal) [inline app/templates/components/events/view/overview/event-sponsors.hbs:9]: Why the need for inline styles?

File: app/controllers/events/view/index.js [hunk 2]
```diff
@@ -27,45 +26,36 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
+  // }
 
-  @action
-  editSponsor() {
-    this.transitionToRoute('events.view.edit.sponsors');
-  }
+  // @action
+  // editSponsor() {
+  //   this.transitionToRoute('events.view.edit.sponsors');
+  // }
```
Comment (iamareebjamal) [inline app/controllers/events/view/index.js:59]: Delete the commented lines please

**Comment by iamareebjamal** [comment]
@maze-runnar Please check the comments from @mariobehling and me about spacing and inline styles. Please try to reduce request review cycles

**Comment by iamareebjamal** [comment]
I know it is for overriding CSS, but it shouldn't be needed

**Comment by iamareebjamal** [comment]
Code looks good to me. Please resolve conflicts

**Comment by mariobehling** [comment]
Let's try to standardize the design.

* Please make the table width the same width as the table in the above box.
* Please use the entire width of the table
   * expand the width of the sponsor names
   * move Type and Level to the right
* Reduce the space of the previous/next buttons to the above table

Screenshot from 2020-11-06 22-19-11

**Comment by mariobehling** [comment]
I see the non-functional search box we already got rid of is back. It seems like you are working on different versions, cause I do not see it in your screenshot.

**Comment by maze-runnar** [comment]
> * Please make the table width the same width as the table in the above box.

@iamareebjamal i have already used 15 wide column class, now for this I have to add `width` css.

**Comment by iamareebjamal** [comment]
Go ahead

**Comment by maze-runnar** [comment]
> Reduce the space of the previous/next buttons to the above table

@iamareebjamal  what about this? the space is kind of fix for every table.

**Comment by maze-runnar** [comment]
- [ ] this is remaining to do.But it will change it in every place
scrnli_11_7_2020_6-14-44 PM

**Comment by iamareebjamal** [comment]
Yes, reduce it for every table please

**Comment by iamareebjamal** [comment]
Screenshot? Also I think the only thing which was needing change was pagination padding, what effect does changing table padding have in other sections of the site?

**Comment by maze-runnar** [comment]
scrnli_11_8_2020_2-09-55 PM
scrnli_11_8_2020_2-09-06 PM

**Comment by mariobehling** [comment]
According to your screenshots you are working with outdated code, which is confusing for me when I review it. So, I am updating the branch.

**Review by mariobehling — CHANGES_REQUESTED** [review]
There are few small issues. Maybe it is difficult for you to see it, but usually from a design perspective across all boxes the spacing of the heading and text should be the same. Below table spacing should be the same as the spacing of the heading. I tried to show it in my previous screenshot. I do not know how to make it clearer.

Could you also please do the following, please:
* Delete the column "Level" (sorry for the addition, but looking over it carefully, it does not really seem necessary here)
* Make the "Name" column wider. We have lots of space left on the right-hand side.
* Ensure the list really only shows the number of sponsors that are chosen: Here it shows "10", but 11 sponsors are shown in the box. There is an error somewhere.

Screenshot from 2020-11-10 13-17-35

**Comment by iamareebjamal** [comment]
@mariobehling Are you talking about the space between the navigation panel and the end of the table? That is part of the table UI and independent of the sponsors section. The space was already reduced and can be reduced further, but it'll stay independent of the space above the table, and for example, look different in other parts of the UI. So, we can make it look equal here but then it'll make it look unequal in other parts of the UI wherever the table is used, so it's a question of trade-off and we can only use a constant spacing in the component.

About the 11 sponsors, that is another issue but can be taken in this as well, but it is not a UI change mentioned in the issue or something which broke due to these changes.

**Comment by mariobehling** [comment]
@iamareebjamal I will file a separate issue later about standardization of design on dashboard. Thanks for taking in things that were not in the original issue! Appreciate it.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**